draft-ietf-rtcweb-jsep.xml

```diff
@@ -5886,6 +5886,12 @@ a=msid:-
 
           <t>If the packet is of type APP, deliver a copy of the packet
           to each "m=" line.</t>
+
+          <t> Some multiparty RTP topologies result in endoints
+          receiving RTCP reports for an SSRC that is not associated
+          with the endpoint that received the RTCP packet. These
+          packets MAY be discarded. </t>
```

Comment:
I wrote it that way so it allows the basic stupid implementation to just drop the data. However, a smart implementation that do want to monitor the current performance of the path between two session participants can do this. At least one of the old MBONE tools had a dashbord that presented also thrid party statistics.